tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners/Program.cs

```diff
@@ -10,26 +11,32 @@ class Program
         /// Retrieves codeowners information for specific section of the repo
         /// </summary>
         /// <param name="targetDirectory">The directory whose information is to be retrieved</param>
-        /// <param name="rootDirectory">The root of the repo or $(Build.SourcesDirectory) on DevOps</param>
-        /// <param name="vsoOwningUsers">Variable for setting user aliases</param>
-        /// <returns></returns>
+        /// <param name="codeOwnerFilePath">The path of CODEOWNERS file in repo</param>
+        /// <returns>Exit code</returns>
 
-        public static void Main(
+        public static int Main(
             string targetDirectory,
```

Comment:
Let's change the order of these parameters so the codeowner file path is first to make it consistent with other places.

Reply:
It is the same order as original code. Which place have codeowner file path first?

Reply:
This is a new tool that isn't really used by anything yet so I'd suggest codeowner file path first then the target.

Reply:
Also we should probably make these names match that of the PS script so we have those consistent as well. If we use target we should use that everywhere.